pkgs/by-name/qb/qbittorrent-exporter/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGo125Module,
+  nix-update-script,
+}:
+buildGo125Module rec {
+  pname = "qbittorrent-exporter";
+  version = "1.11.0";
+
+  src = fetchFromGitHub {
+    owner = "martabal";
+    repo = pname;
+    rev = "v${version}";
```

```suggestion
    tag = "v${finalAttrs.version}";
```
Let's use tag if we're not referencing a commit